Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -971,6 +971,11 @@ instance WellOrderingRel.isWellOrder : IsWellOrder α WellOrderingRel :=
 instance IsWellOrder.subtype_nonempty : Nonempty { r // IsWellOrder α r } :=
   ⟨⟨WellOrderingRel, inferInstance⟩⟩
 
+variable (α) in
+theorem exists_wellOrder : ∃ (_ : LinearOrder α), WellFoundedLT α := by
```

Comment:
```suggestion
variable (α) in
/-- The **well-ordering theorem** (or **Zermelo's theorem**): every type has a well-order -/
theorem exists_wellOrder : ∃ (_ : LinearOrder α), WellFoundedLT α := by
```

Reply:
I'm a bit surprised no other theorem had that docstring already.